routes/web.php

```diff
@@ -31,6 +31,8 @@
         // Controllers Within The "App\Http\Controllers\Admin" Namespace
         Route::get('/', 'AdminController@index')->name('admin.index');
         Route::get('/users', 'AdminController@users')->name('admin.users');
+        Route::get('/user/create', 'AdminController@userCreate')->name('admin.user.create');
+        Route::get('/user/edit', 'AdminController@userEdit')->name('admin.user.update');
         Route::get('/blogs', 'AdminController@blogs')->name('admin.blogs');
```

Comment:
content menu.blade.php and in here , are they match each other yet ?

Reply:
Yes, it match